source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -419,6 +452,32 @@ export class Carousel extends Container {
     }
 }
 
+export class CarouselEvent {
+    type : Enums.CarouselEventType;
+    carouselId : string | undefined;
+    activeCarouselPageId : string | undefined;
+    activeCarouselPageIndex : number | undefined;
```

Comment:
Is there anyone requesting pageId and index? if not, we can remove it and add it late

Reply:
those are needed so that the users can figure out which page is being used and not used..